demos/simple/index.html

```diff
@@ -32,15 +32,16 @@
     <script src="../../build/js/live-editor.tooltips.js"></script>
     <script src="../../build/js/live-editor.ui.js"></script>
     <script src="../../build/js/live-editor.editor_ace.js"></script>
-    <script src="../../build/js/live-editor.debugger.js"></script>
+    <!--<script src="../../build/js/live-editor.debugger.js"></script>-->
```

Comment:
I want to conditional load these files.  I'm disabling for now and will add conditional loading in a separate pull request.